coreUtilities/src/main/java/org/opensearch/migrations/transformation/rules/IndexMappingTypeRemoval.java

```diff
@@ -0,0 +1,54 @@
+package org.opensearch.migrations.transformation.rules;
+
+import java.util.Map.Entry;
+
+import org.opensearch.migrations.transformation.CanApplyResult;
+import org.opensearch.migrations.transformation.CanApplyResult.Unsupported;
+import org.opensearch.migrations.transformation.TransformationRule;
+import org.opensearch.migrations.transformation.entity.Index;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Supports transformation of the Index Mapping types that were changed from mutliple types to a single type between ES 6 to 7
```

Comment:
Can we get a full example transformation here?  I'm having a hard time understanding the intent even after reading through the code and unit tests a couple times.